lib/www/clarification.php

```diff
@@ -329,7 +329,7 @@ function confirmClar() {
 		if ( ! $respid ) {
 			$teams = $DB->q('KEYVALUETABLE SELECT teamid, name
 			                 FROM team
-			                 ORDER BY categoryid ASC, team.name COLLATE utf8_general_ci ASC');
+			                 ORDER BY categoryid ASC, team.name COLLATE '. DJ_CHARACTER_SET_MYSQL . '_general_ci ASC');
```

Comment:
I would make the entire collation a new constant, defined next to `DJ_CHARACTER_SET_MYSQL` and not be 'collating' the collation-string deep in the code.